### PR TITLE
minor changes to README.md, env requirements

### PR DESCRIPTION
**Pull request by JamesKunstle**
@cdolfi This updates docs to reflect some changes to the environment requirements and the folder structure. I'm 50/50 on sketching out the caching system before we update the augur replica. What are your thoughts?

### CODE REVIEW

**Comment by cdolfi** [comment]
@JamesKunstle I believe we should bring the docs to date to the current structure, especially with the amount of people trying 8Knot for the first time. I would say if there is portions of it you think will be changed soon I wouldnt go into too much detail

**Comment by JamesKunstle** [comment]
@cdolfi Which structure are you specifically describing? The caching architecture or the file layout structure?

**Comment by cdolfi** [comment]
@JamesKunstle Structure is not actually the right word to use here my b. My only concern at this point is that our readme has the updated information on how to locally host and run an 8knot instance. On the description of the caching, maybe just a little burb with a 1 sentance-ish description of what we have now and that it will be updated soon with the new changes

**Comment by JamesKunstle** [comment]
@cdolfi Okay, I updated the docs to be clearer about how to configure and deploy the application, and a comment about our caching likely changing in the near future. What do you think?

File: README.md [hunk 1]
```diff
@@ -120,14 +136,28 @@ If you would prefer to look at our most up-to-date work, please check out the fo
 
 ---
 
-## Local Development
+## Architectural Comment
 
-We've tried to make it as easy as possible to go from fork/clone to a running instance.
+Our demonstration application run in us-west2, while the Augur database that we use is in Columbus, MO. To improve user experience,
```
Comment (cdolfi) [inline README.md:141]: Outside of Red Hat I dont think people will know what "us-west2" means

**Review by cdolfi — CHANGES_REQUESTED** [review]
one small thing and then itll be good to go from my side!

File: README.md (hunk 1 above)
Comment (JamesKunstle) [reply README.md:141]: Oh, that's an AWS zone. I'll clarify but that's a conventional AWS region that's well-known.

Comment (JamesKunstle) [reply README.md]: updated!

**Review by cdolfi — APPROVED** [review]
lgtm

**Pull request merged**